### ISSUE TITLE
M0 training result

### ISSUE THREAD

**Issue by BHC1205** [Sense-GVT/Fast-BEV#36]
This accuracy does not match with the paper, the feeling is much higher, is it normal？

fastbev_m0_r18_s256x704_v200x200x4_c192_d2_f4.py
mAP: 0.3706
mATE: 0.7414
mASE: 0.2416
mAOE: 0.6121
mAVE: 0.6098
mAAE: 0.2628
NDS: 0.4386
Eval time: 177.4s

Per-class results:
Object Class	AP	ATE	ASE	AOE	AVE	AAE
car	0.566	0.476	0.146	0.254	0.702	0.276
truck	0.384	0.659	0.189	0.289	0.584	0.275
bus	0.408	0.676	0.194	0.211	1.153	0.298
trailer	0.144	1.181	0.207	0.437	0.391	0.198
construction_vehicle	0.179	0.945	0.381	1.073	0.144	0.326
pedestrian	0.420	0.729	0.275	1.029	0.628	0.397
motorcycle	0.338	0.747	0.222	1.027	1.031	0.291
bicycle	0.304	0.640	0.262	1.072	0.244	0.040
traffic_cone	0.506	0.624	0.317	nan	nan	nan
barrier	0.458	0.739	0.223	0.117	nan	nan

**Comment by Ignite616** [comment]
Hello, how to use the `cascade_ mask_ rcnn_ r18_ fpn_ coco-mstrain_ 3x_ 20e_ nuim_ bbox_ mAP_ 0.5110_ segm_ mAP_ 0.4070.pth ` provided by fast-bev during training .   
I run the command  `python tools/train.py configs/fastbev/exp/paper/fastbev_m1_r18_s320x880_v200x200x4_c192_d2_f4.py   --load-from cascade_mask_rcnn_r18_fpn_coco-mstrain_3x_20e_nuim_bbox_mAP_0.5110_segm_mAP_0.4070.pth`
when I load the pre-training cascade_ mask_ rcnn model, there will be some warnings: size mismatch for neck.

Let me ask you what GPUs you use and how long you have trained. I use two TITAN RTX, and set the batch size of each GPU is 8. I Spent 21 hours training to epoch[1] [4530/8007].  At this speed, I need 31 days to train 20 epoches. It seems that the training is very slow.

**Comment by Terencedu** [comment]
@Ignite616 Hi, I also meet the size mismatch problem when train "cascade_mask_rcnn_r18_fpn_coco-mstrain_3x_20e_nuim_bbox_mAP_0.5110_segm_mAP_0.4070", did you solve it?
